internal/store/automation.go

```diff
@@ -30,20 +30,35 @@ type AutomationUpdater interface {
 	UpdateAutomationConfig(string, *om.AutomationConfig) error
 }
 
+type AutomationStatusGetter interface {
+	GetAutomationConfigStatus(string) (*om.AutomationStatus, error)
+}
+
 type AllClusterLister interface {
 	ListAllClustersProjects() (*om.AllClustersProjects, error)
 }
 
 type AutomationStore interface {
 	AutomationGetter
 	AutomationUpdater
+	AutomationStatusGetter
 }
 
 type CloudManagerClustersLister interface {
 	AutomationGetter
 	AllClusterLister
 }
 
+func (s *Store) GetAutomationConfigStatus(projectID string) (*om.AutomationStatus, error) {
```

Comment:
this could be just `GetAutomationStatus` but not a blocker since there's nothing wrong (that I can see) on this PR 👍